src/main/java/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.java

```diff
@@ -62,7 +61,7 @@ public interface StudyViewMapper {
 
     List<GenomicDataCountItem> getCNACounts(StudyViewFilterHelper studyViewFilterHelper, List<GenomicDataFilter> genomicDataFilters);
 
-    Map<String, Integer> getMutationCounts(StudyViewFilterHelper studyViewFilterHelper, GenomicDataFilter genomicDataFilter);
-
     List<GenomicDataCountItem> getMutationCountsByType(StudyViewFilterHelper studyViewFilterHelper, List<GenomicDataFilter> genomicDataFilters);
+
+    AlterationCountByGene getMutatedGene(StudyViewFilterHelper studyViewFilterHelper, AlterationFilterHelper alterationFilterHelper, String specificHugoGeneSymbol);
```

Comment:
Am I missing this? I don't see it defined in the xml?

Reply:
Oh sorry I must forgot to remove it